x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/math/MathProcessor.java

```diff
@@ -77,7 +76,7 @@
         LOG10(Math::log10),
         PI(() -> Math.PI),
         RADIANS(Math::toRadians),
-        RANDOM((Object l) -> l != null ? new Random(((Number) l).longValue()).nextDouble() : Randomness.get().nextDouble(), true),
```

Comment:
This was the only place where the `true` value for `nullAware` was used. Maybe it's worth adjusting the constructors as well.

Reply:
good point, I removed the ctor 👍